packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
```

Comment:
Where do these types come from? As in, where is the list of category groups defined? It would be good to reference that here.

Reply:
This is mapping the output for the PageSpeed Insights API. It's not comprehensive, but since there's no TypeScript mapping yet, I tried mapping more than needed.

Reply:
Could you add that as a comment somewhere? Ideally also with a few example URLs that you used to get an idea of the response types?

Reply:
Added a link